be/src/storage/lake/lake_persistent_index.cpp

```diff
@@ -248,23 +248,49 @@ Status LakePersistentIndex::try_replace(size_t n, const Slice* keys, const Index
     return Status::OK();
 }
 
-std::unique_ptr<sstable::Iterator> LakePersistentIndex::prepare_merging_iterator() {
+Status LakePersistentIndex::prepare_merging_iterator(
+        const TabletMetadata& metadata, TxnLogPB* txn_log,
+        std::vector<std::shared_ptr<PersistentIndexSstable>>* merge_sstables,
```

Comment:
`std::vector<std::shared_ptr<PersistentIndexSstable>>* cloned_sstables`?

Reply:
Change `cloned_sstables` to `merge_sstables`.